python/tests/test_approximate_nearest_neighbors.py

```diff
@@ -808,9 +839,110 @@ def test_ivfflat_wide_matrix(
     data_shape: Tuple[int, int],
     data_type: np.dtype,
 ) -> None:
+    """
+    It seems adding a column with df.withColumn can be very slow, if df already has many columns (e.g. 3000).
```

Comment:
This comment would make more sense closer to the actual withColumn .

Reply:
Revised the comment in utils.py.